src/components/SwipeableView/index.native.js

```diff
@@ -10,35 +10,32 @@ const propTypes = {
     onSwipeDown: PropTypes.func.isRequired,
 };
 
-class SwipeableView extends PureComponent {
-    constructor(props) {
-        super(props);
-
-        const minimumPixelDistance = CONST.COMPOSER_MAX_HEIGHT;
-        this.oldY = 0;
-        this.panResponder = PanResponder.create({
+function SwipeableView({children, onSwipeDown}) {
```

Comment:
We should just get param `props` instead of spreading the params directly. It will be consistent with all other places in our codebase and it would be easy to understand from where the value came from when using

@ojasjadhav2